ga/networkedgeneticalgorithm.py

```diff
@@ -24,7 +24,7 @@
 from distributed.joblib import DaskDistributedBackend
 
 creator.create("FitnessMax", base.Fitness, weights=(1.0,))
-creator.create("Individual", numpy.ndarray, fitness=creator.FitnessMax)
+creator.create("Individual", numpy.ndarray, fitness=creator.FitnessMax)  # @UndefinedVariable
```

Comment:
What do you mean by this?

Reply:
It's a thing PyDev puts so it can ignore the fact that _FitnessMax_ is not a defined field in the _creator_ module. Otherwise it throws me an error (it can probably run, but marks it red and is annoying).
Basically PyDev has a problem with stuff added dynamically to the list of global variables using "global()" (which is what "creator.create" does).

Reply:
It'd be nice to not have IDE specific code in the project if it can be avoided. Especially if that IDE is the unholy Eclipse ;)

Reply:
Yes, I understand, but it's just a small random comment, not active code, so I don't think there's any harm or ugliness. And it is even informative, since it is a (statically) undefined variable and any IDE will not be able to suggest it to you, you just have to know that it's there... (I didn't know you could do a thing like that, but anything's possibly in Python).

btw. what's wrong with Eclipse? :P It's actually great for Python, which is very convenient if you use it for everything else too...